### PR TITLE
Container: set name when known

### PR DESCRIPTION
**Pull request by dhodovsk**

### CODE REVIEW

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/HbXw](https://da.gd/HbXw)__
* Comment: __See details for logs__

File: conu/backend/docker/image.py [hunk 1]
```diff
@@ -283,6 +284,11 @@ def callback():
             return subprocess.Popen(run_command_instance.build(), **popen_params)
         container_id, popen_instance = self._run_container(run_command_instance, callback)
 
+        actual_name = self.d.inspect_container(container_id)['Name'][1:]
+        if container_name and container_name != actual_name:
+            raise ConuException("Container name  is not set to desired value")
```
Comment (TomasTomecek) [inline conu/backend/docker/image.py]: nit: could we display container_name and actual_name here? there is also double space in the message

File: conu/backend/docker/image.py [hunk 2]
```diff
@@ -247,7 +247,8 @@ def callback():
                 raise ConuException("Container exited with an error: %s" % ex.returncode)
         container_id, _ = self._run_container(run_command_instance, callback)
 
-        return DockerContainer(self, container_id)
+        container_name = self.d.inspect_container(container_id)['Name'][1:]
```
Comment (TomasTomecek) [inline conu/backend/docker/image.py]: That leading `/` slash is so retarded in the name, I don't understand it -_-

**Comment by psklenar** [comment]
## jenkins/superCI comment
* Status: success, return code = __0__ 
* Result url: __[https://da.gd/wppJN](https://da.gd/wppJN)__
* Comment: __See details for logs__

**Pull request merged**